app/src/main/java/com/example/arc_exapmle/activity/AddNoteKtActivity.kt

```diff
@@ -97,35 +91,33 @@ class AddNoteKtActivity : AppCompatActivity() {
     override fun onResume() {
         super.onResume()
 
-        addNoteKtActivityViewModel.titleMutableLiveData.observe(this , Observer {
+        addNoteKtActivityViewModel.titleMutableLiveData.observe(this, Observer {
 
             editTextTitle.error = it
 
         })
 
-        addNoteKtActivityViewModel.descriptionMutableLiveData.observe(this , Observer {
+        addNoteKtActivityViewModel.descriptionMutableLiveData.observe(this, Observer {
 
             editTextDescription.error = it
 
         })
 
-        addNoteKtActivityViewModel.toastMutableLiveData.observe(this , Observer {
+        addNoteKtActivityViewModel.toastMutableLiveData.observe(this, Observer {
 
-            Toast.makeText(this , it , Toast.LENGTH_SHORT).show()
+            Toast.makeText(this, it, Toast.LENGTH_SHORT).show()
 
         })
 
-        addNoteKtActivityViewModel.onSuccessMutableLiveData.observe(this , Observer {
+        addNoteKtActivityViewModel.onSuccessMutableLiveData.observe(this, Observer {
 
-            if(it){
+            if (it) {
```

Comment:
mage az singleLiveData estefade nemikoni inja?
pas in if baraye chie?

Reply:
pakidamesh